docs_test.go

```diff
@@ -20,6 +23,119 @@ func TestToMarkdownFull(t *testing.T) {
 	expectFileContent(t, "testdata/expected-doc-full.md", res)
 }
 
+func TestToTabularMarkdownFull(t *testing.T) {
+	// Given
+	app := testApp()
+
+	// When
+	res, err := app.ToTabularMarkdown("app")
```

Comment:
can you add a test case where you pass in empty string to ToTabularMarkdown ?

Reply:
Done in 05cf3862adae33760cbe3ef7a9551abeacbc07f8 (custom path provided) and 949f45e950f427d93f41286f38707e352480699e (empty)